cli/pcluster/easyconfig.py

```diff
@@ -115,52 +153,47 @@ def ec2_conn(aws_region_name):
     return ec2
 
 
+def extract_tag_from_resource(resource, tag_name):
+    tags = resource.get("Tags")
+    if tags:
+        for item in tags:
+            if item["Key"] == tag_name:
+                return item["Value"]
+
+
+def list_resource(resource, aws_region_name, resource_name, resource_id_name):
```

Comment:
It would be useful to have a comment for this function

Reply:
Should be a private function